en/env.rst

```diff
@@ -378,3 +383,18 @@ The following table summarizes the ports, based on the listening processes, used
 +--------------------------+--------------+----------------+--------------------------+
 
 *   The port used when the CUBRID Manager client accesses the CUBRID Manager server process is **cm_port** of the cm.conf. The default value is 8001.
+
+Ports for CUBRID Java Stored Procedure Server
+---------------------------------------------
+
+The following table summarizes the ports, based on the listening processes, used for CUBRID Java Stored Procedure (Java SP) server. The ports are identical regardless of the OS type.
+
++---------------+--------------+----------------------------+--------------------------+
+| Listener      | Requester    | Port                       | Firewall Port Setting    |
++===============+==============+============================+==========================+
+| cub_javasp    | CAS          | java_stored_procedure_port | Open                     |
++---------------+--------------+----------------------------+--------------------------+
+
+*   The port used when the CAS relays between Java SP server (cub_javasp) and cub_server, which CAS receives a call of the java stored procedure from cub_server and then CAS passed the call to the CUBRID Java SP server process through **java_stored_procedure_port** of cubrid.conf.
```

Comment:
propose: The port used --> The port is used

Reply:
Thank you, I'll fix it